qiskit/transpiler/passes/scheduling/calibration_creators.py

```diff
@@ -269,27 +278,23 @@ class RZXCalibrationBuilderNoEcho(RZXCalibrationBuilder):
     """
     Creates calibrations for RZXGate(theta) by stretching and compressing
     Gaussian square pulses in the CX gate.
-
-    The ``RZXCalibrationBuilderNoEcho`` is a variation of the
-    :class:`~qiskit.transpiler.passes.RZXCalibrationBuilder` pass
-    that creates calibrations for the cross-resonance pulses without inserting
+    The RZXCalibrationBuilderNoEcho is a variation of the RZXCalibrationBuilder
+    as it creates calibrations for the cross-resonance pulses without inserting
     the echo pulses in the pulse schedule. This enables exposing the echo in
     the cross-resonance sequence as gates so that the transpiler can simplify them.
-    The ``RZXCalibrationBuilderNoEcho`` only supports the hardware-native direction
+    The RZXCalibrationBuilderNoEcho only supports the hardware-native direction
     of the CX gate.
     """
 
     @staticmethod
     def _filter_control(inst: (int, Union["Schedule", Instruction])) -> bool:
         """
         Looks for Gaussian square pulses applied to control channels.
-
```

Comment:
Can we keep the spaces here to not make the docs look too dense?

Reply:
See a484c9c.